locales/en-US/editor.properties

```diff
@@ -218,6 +218,15 @@ PIXEL_UNIT=pixels
 PERCENTAGE_UNIT=percentage
 EM_UNIT=em font unit
 
+# extensions/default/InlinePaddingEditor
+# "All Sides" is an option that can be selected for padding in case when users want to change padding value for all the sides at once
+ALL_SIDE=All Sides
+# "Individual Sides" is an option that can be selected for padding in case when users want to change the padding for each side indiviually 
```

Comment:
make this a single comment otherwise our l10n platform won't pick it up.

Reply:
Sure !